### PR TITLE
docs(grammar): fix using grammarly

### PR DESCRIPTION
**Pull request by krishguptadev**
I recently opened up a similar pr to our [hacktoberfest-practice repo](https://github.com/EddieHubCommunity/hacktoberfest-practice/pull/777), This PR has the same purpose, to fix Grammarly issues and alexjs issues.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/759"></a>

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

File: Contributing.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 [![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
 
 - Take a look at the existing [Issues](https://github.com/EddieHubCommunity/LinkFree/issues) or [create a new issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose)!
-- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/fork), create a branch for any issue that you are working on and commit your work.
+- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/fork). Then, create a branch for any issue that you are working over. Finally, commit your work.
```
Comment (emmalearnscode) [inline Contributing.md:7]: It should be on, not over

File: Contributing.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ git push origin YourBranchName
 
 ## 💥 Issues
 
-For major changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
+For large/discussable changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```
Comment (emmalearnscode) [inline Contributing.md:96]: I think major sounds better here

File: README.md [hunk 3]
```diff
@@ -9,10 +9,10 @@
 
 > A note for Hacktoberfest Participants:
 >
-> Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT be counted for Hacktoberfest.
+> Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT count for Hacktoberfest.
```
Comment (emmalearnscode) [inline README.md:12]: Be counted works better here

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

File: Contributing.md (hunk 1 above)
Comment (krishguptadev) [reply Contributing.md:7]:
```suggestion
- [Fork the Repo](https://github.com/EddieHubCommunity/LinkFree/fork). Then, create a branch for any issue that you are working on. Finally, commit your work.
```

File: README.md (hunk 3 above)
Comment (krishguptadev) [reply README.md:12]: It is incorrect use of passive voice tho.

Comment (emmalearnscode) [reply README.md:12]: I don’t think it’s wrong. The vote will be counted by some other subject so will be sounds better. Remember grammarly is a computer program that can’t always detect nuances so it’s not always correct either. As a native English speaker sometimes things just sound wrong to the ear, like it’s not natural and for me this is one of those times. In this case it’s not a massive issue but be wary in believing everything grammarly tells you ;-)

File: Contributing.md (hunk 2 above)
Comment (emmalearnscode) [reply Contributing.md:96]: Again, large/discussable really doesn’t sound right here, I’m not sure discussable is even a real word

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

File: Contributing.md (hunk 2 above)
Comment (krishguptadev) [reply Contributing.md:96]:
```suggestion
For major changes and controversial/opinionated changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```

File: README.md (hunk 3 above)
Comment (krishguptadev) [reply README.md:12]:
```suggestion
> Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT be counted for Hacktoberfest.
```

File: Contributing.md (hunk 2 above)
Comment (Vyvy-vi) [reply Contributing.md]: I think we can probably mention -> 
```
For discussing changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```

Comment (Vyvy-vi) [reply Contributing.md]: Because regardless, we encourage people to open an issue first.

File: README.md [hunk 4]
```diff
@@ -123,11 +126,11 @@ Your `avatar` URL should take the format of `https://github.com/<yourusername>.p
 
 ## 🛡️ License
 
-This project is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.
+Linkfree is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.
```
Comment (Panquesito7) [inline README.md:129]:
```suggestion
LinkFree is licensed under the MIT License - see the [`LICENSE`](LICENSE) file for details.
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -105,8 +105,11 @@ Optional fields: `links` and `milestones`
 }
 ```
 
-Your URL will be `http://linkfree.eddiehub.org/<yourusername>`. For example: <http://linkfree.eddiehub.org/eddiejaoude>\
-Your `avatar` URL should take the format of `https://github.com/<yourusername>.png`.
+Your URL will be `http://linkfree.eddiehub.org/<yourusername>`
+
+For example <http://linkfree.eddiehub.org/eddiejaoude>\
```
Comment (Panquesito7) [inline README.md:110]:
```suggestion
For example: <http://linkfree.eddiehub.org/eddiejaoude>\
```

**Review by Vyvy-vi — APPROVED** [review]
LGTM!

File: Contributing.md [hunk 6]
```diff
@@ -93,7 +93,7 @@ git push origin YourBranchName
 
 ## 💥 Issues
 
-For major changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
+For discussing changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```
Comment (emmalearnscode) [inline Contributing.md:96]: In order to discuss changes

Comment (krishguptadev) [reply Contributing.md:96]:
```suggestion
In discussing changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```

**Review by Panquesito7 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Review by Shivansh-yadav13 — APPROVED** [review]

File: Contributing.md [hunk 7]
```diff
@@ -93,7 +93,7 @@ git push origin YourBranchName
 
 ## 💥 Issues
 
-For major changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
+In discussing changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```
[the remark this answers is not in the document]
Comment (krishguptadev) [reply Contributing.md:96]:
```suggestion
In order to discuss changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```

**Review by Vyvy-vi — APPROVED** [review]
lgtm. Thanks for contributing to LinkFree!

**Pull request merged**